README.md

```diff
@@ -0,0 +1,121 @@
+# Solid Notification Client 
+
+This library implements the disovery and subscription to a Notication Channel following the [Solid Notifications Protocol](https://solidproject.org/TR/notifications-protocol).
+
+## Discovery
+
+A first step for having Solid Notifications is having a resource of interest to which you want to subscribe, we will call this resource the **topic**.
+To [discover](https://solidproject.org/TR/notifications-protocol#discovery) whether the topic supports a certain [**channel type**](https://solid.github.io/notifications/protocol#notification-channel-types), you can use the **DiscoveryClient**. (A list of channel types can be found [here](https://github.com/solid/notifications#notificatoin-channel-types))
```

Comment:
```suggestion
To [discover](https://solidproject.org/TR/notifications-protocol#discovery) whether the topic supports a certain [**channel type**](https://solid.github.io/notifications/protocol#notification-channel-types), you can use the **DiscoveryClient**. (A list of channel types can be found [here](https://solidproject.org/TR/#notification-channel-type-registry))
```

This link is a little bit more official, at least until Solid WG gets created and picks up the notification work.